<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreboardMatchDetailsUI</class>
 <widget class="QWidget" name="ScoreboardMatchDetailsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="scoreboardLayout"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Match Details</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="player1NameLabel_2">
          <property name="text">
           <string>Player 1:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="player1NameLabel"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="player2NameLabel_2">
          <property name="text">
           <string>Player 2:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="player2NameLabel"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="player1ScoreLabel_2">
          <property name="text">
           <string>Score 1:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="player1ScoreLabel"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="player2ScoreLabel_2">
          <property name="text">
           <string>Score 2:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="player2ScoreLabel"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="matchComboBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>